android/modules/ui/src/java/ti/modules/titanium/ui/TableViewProxy.java

```diff
@@ -137,6 +139,13 @@ public void releaseViews()
 	@Override
 	public void release()
 	{
+		// Release search bar proxy if there is one
+		if (getTableView().getNativeView() instanceof RelativeLayout) {
```

Comment:
Can getTableView() return null?
Also, assign a temp var to hold result of `getTableView().getNativeView()` since you're calling it twice consecutively.

Reply:
I think not. It either returns the already available table view or creates one. But maybe checking for having PROPERTY_SEARCH assigned a value would be a better solution. We don't really care if the search view was drawn on the screen or not before releasing the proxy it holds. I am on it.